src/audio.c

```diff
@@ -185,8 +194,10 @@ static void audio_callback()
  *            The frequency in Hz to play back samples at
  * @param[in] numbuffers
  *            The number of buffers to allocate internally
+ * @param[in] fill_buffer_callback
+ *            A function to be called when more sample data is needed
  */
-void audio_init(const int frequency, int numbuffers)
+void audio_init(const int frequency, int numbuffers, audio_fill_buffer_callback fill_buffer_callback)
```

Comment:
I don't see any updates to the examples. I forget, did any of the examples show off audio? If so, they won't compile anymore.